README.md

```diff
@@ -137,38 +162,142 @@ generate RBIs from gems
+
+To help you get started, you can use the command `tapioca require` to auto-populate the contents of the `sorbet/tapioca/require.rb` file with all the requires found in your application:
+
+```sh
+$ bin/tapioca require
+
+Compiling sorbet/tapioca/require.rb, this may take a few seconds... Done
+
+All requires from this application have been written to sorbet/tapioca/require.rb.
+Please review changes and commit them, then run `bin/tapioca gem`.
+```
+
+Once the file is generated, you should review it, remove all unnecessary requires and commit it.
+
+#### Excluding a gem from RBI generation
+
+It may be useful to exclude some gems from the generation process. For example for gems that are in Bundle's debug group or gems of which the contents are dependent on the architecture they are loaded on.
+
+To do so you can pass the list of gems you want to exclude in the command line with the `--exclude` option:
+
+```sh
+$ bin/tapioca gems --exclude gemA gemB
+```
+
+Or through the configuration file:
+
+```yaml
+gem:
+  exclude:
+    - gemA
+    - gemB
+```
+
+#### Changing the strictness level of the RBI for a gem
+
+By default, all RBI files for gems are generated with the [strictness level](https://sorbet.org/docs/static#file-level-granularity-strictness-levels) `typed: true`. Sometimes, this strictness level can create type-checking errors when a gem contains definitions that conflict with [Sorbet internal definitions for Ruby core and standard library](https://sorbet.org/docs/faq#it-looks-like-sorbets-types-for-the-stdlib-are-wrong).
+
+Tapioca comes with an automatic detection (option `--auto-strictness` enabled by default) of such cases and will switch the strictness level to `typed: false` in RBI files containing conflicts with the core and standard library definitions. If is nonetheless possible to manually switch the strictness level for a gem using the `--typed-overrides` option:
+
+```sh
+$ bin/tapioca gems --typed-overrides gemA:false gemB:false
+```
+
+Or through the configuration file:
+
+```yaml
+gem:
+  typed_overrides:
+    gemA: "false"
+    gemB: "false"
 ```
-<!-- END_HELP_COMMAND_TODO -->
 
-### Generate DSL RBI files
+#### Keeping RBI files for gems up-to-date
 
-Command: `tapioca dsl [constant...]`
+To ensure all RBI files for gems are up-to-date with the latest changes in your `Gemfile.lock`, Tapioca provides a `--verify` option:
+
+```sh
+$ bin/tapioca gems --verify
+
+Checking for out-of-date RBIs...
+
+Nothing to do, all RBIs are up-to-date.
+```
+
+This option can be used on CI to make sure the RBI files are always up-to-date and ensure accurate type checking. **Warning**: doing so will break your normal Rubocop workflow as every pull-request opened to bump a gem version will fail CI since the RBI will be out-of-date and will require you to manually run `bin/tapioca gems` to update them.
```

Comment:
```suggestion
This option can be used on CI to make sure the RBI files are always up-to-date and ensure accurate type checking. **Warning**: doing so will break your normal Dependabot workflow as every pull-request opened to bump a gem version will fail CI since the RBI will be out-of-date and will require you to manually run `bin/tapioca gems` to update them.
```